_stbt/core.py

```diff
@@ -2283,12 +2296,11 @@ def _match_template(image, template, method, roi_mask, level, imwrite):
         rois = [  # Initial region of interest: The whole image.
             _Rect(0, 0, matches_heatmap.shape[1], matches_heatmap.shape[0])]
     else:
-        contours, _ = cv2.findContours(
-            roi_mask, cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_NONE)
+        contours = _find_contours(
+            image=roi_mask, mode=cv2.RETR_EXTERNAL,
+            method=cv2.CHAIN_APPROX_NONE)
         rois = [
             _Rect(*cv2.boundingRect(x))
-            # findContours ignores 1-pixel border of the image
-            .shift(Position(-1, -1)).expand(_Size(2, 2))
```

Comment:
TODO: I think I've preserved the `.shift(Position(-1, -1)` with the new implementation by adding `offset=(1, 1)`, but I've thrown away the expand bit.  I'll need to put it back to preserve the same behaviour, but first I need to figure out how it's changed between opencv 2 and 3.